business/test_util.go

```diff
@@ -722,6 +726,16 @@ func FakePodLogsProxy() *kubernetes.PodLogs {
 	}
 }
 
+func FakePodLogsZtunnel() *kubernetes.PodLogs {
+	content, err := readFile("../tests/data/logs/ztunnel.log")
+	if err != nil {
+		log.Errorf("Error reading logs file: %s", err.Error())
+	}
```

Comment:
nit: you should be able to just embed the file like this: https://github.com/kiali/kiali/blob/6bbef003ec097d104a9120588841eada9937fd75/kubernetes/client_factory_test.go#L28-L29